### DISCUSSION TITLE
官方文档 excel response导出有瑕疵

### DISCUSSION THREAD

**Discussion by MaLinYu** [looly/hutool#2582] [General]
hitool 官方文档
--[Excel生成-ExcelWriter]
--[6. 写出到客户端下载（写出到Servlet）]
--2-写出xlsx 
 少了这句代码  ServletOutputStream out=response.getOutputStream();   不可以直接复制粘贴,不算是bug吧.反馈一下,大佬看我
 文件导出设置中文名
 response.setHeader("Content-Disposition","attachment;filename=" + new String("文件名.xlsx".getBytes("gb2312"),"ISO8859-1"));

**Comment by looly** [comment]
文档最近修正~~感谢。
